CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
+### Changed
+
+* We changed the default configuration to be more user convenient:
+  * Outlook does not need to be configured to convert emails: `DEFAULT_CONVERT_TO_PLAIN` is `True` (instead of `False`)
+  * QuoteFixMacro does not require to redconfigure the signatures any more: `DEFAULT_USE_QUOTING_TEMPLATE` is `True` (instead of `False`)
+  * The newly opened window is made as wide as required to display the wrapping is it will be when the email being sent: `DEFAULT_USE_SOFTWRAP` is `True` (instead of `False`)
```

Comment:
```suggestion
  * QuoteFixMacro does not require to reconfigure the signatures any more: `DEFAULT_USE_QUOTING_TEMPLATE` is `True` (instead of `False`)
  * The newly opened window is made as wide as required to display the wrapping as it will be when the email is being sent: `DEFAULT_USE_SOFTWRAP` is `True` (instead of `False`)
```

Could you also change typo of "eease" in line 31, please.